api/bindings.h

```diff
@@ -144,14 +144,86 @@ Buffer handle(cache_t *cache,
               uint64_t *gas_used,
               Buffer *err);
 
+Buffer ibc_channel_close(cache_t *cache,
+                         Buffer code_id,
+                         Buffer env,
+                         Buffer msg,
+                         DB db,
+                         GoApi api,
+                         GoQuerier querier,
+                         uint64_t gas_limit,
+                         bool print_debug,
+                         uint64_t *gas_used,
+                         Buffer *err);
+
+Buffer ibc_channel_connect(cache_t *cache,
+                           Buffer code_id,
+                           Buffer env,
+                           Buffer msg,
+                           DB db,
+                           GoApi api,
+                           GoQuerier querier,
+                           uint64_t gas_limit,
+                           bool print_debug,
+                           uint64_t *gas_used,
+                           Buffer *err);
+
+Buffer ibc_channel_open(cache_t *cache,
+                        Buffer code_id,
+                        Buffer env,
+                        Buffer msg,
+                        DB db,
+                        GoApi api,
+                        GoQuerier querier,
+                        uint64_t gas_limit,
+                        bool print_debug,
+                        uint64_t *gas_used,
+                        Buffer *err);
+
+Buffer ibc_packet_ack(cache_t *cache,
+                      Buffer code_id,
+                      Buffer env,
+                      Buffer msg,
+                      DB db,
+                      GoApi api,
+                      GoQuerier querier,
+                      uint64_t gas_limit,
+                      bool print_debug,
+                      uint64_t *gas_used,
+                      Buffer *err);
+
+Buffer ibc_packet_receive(cache_t *cache,
+                          Buffer code_id,
+                          Buffer env,
+                          Buffer msg,
+                          DB db,
+                          GoApi api,
+                          GoQuerier querier,
+                          uint64_t gas_limit,
+                          bool print_debug,
+                          uint64_t *gas_used,
+                          Buffer *err);
+
+Buffer ibc_packet_timeout(cache_t *cache,
+                          Buffer code_id,
+                          Buffer env,
+                          Buffer msg,
+                          DB db,
+                          GoApi api,
+                          GoQuerier querier,
+                          uint64_t gas_limit,
+                          bool print_debug,
```

Comment:
We could move this option from instance level to cache level in cosmwasm to avoid the verbosity for a flexibility we never need (other PR).

Reply:
Sounds like an easy fix